desktopexporter/exporter.go

```diff
@@ -20,7 +20,7 @@ type desktopExporter struct {
 }
 
 func newDesktopExporter(cfg *Config) *desktopExporter {
-	server := server.NewServer(cfg.Endpoint, cfg.DbPath)
+	server := server.NewServer(cfg.Endpoint, cfg.DbPath, cfg.IsDev)
```

Comment:
We're accumulating options here. I think 3 is fine as an arguments list, but if this grows more we should probably define a config object

Reply:
I made `EnvConfig` struct. It holds the environment settings and is just happy to help.